packages/configs/default/index.json

```diff
@@ -5,6 +5,7 @@
       "@parcel/transformer-babel",
       "@parcel/transformer-js"
     ],
+    "manifest.json": ["@parcel/transformer-webext-manifest"],
```

Comment:
This seems dangerous to me. Many things could be called `manifest.json` in theory. I'd say, keep the web extension transformer as an official plugin, but don't include in the default config. That way, if you're writing a web extension, you can opt-in to the processing of `manifest.json`, and it won't affect other apps.

Reply:
So, should I add this logic to `json-transformer`?
```
    if (asset.env.context === 'pwa-manifest') {
      asset.type = 'webmanifest';
    }
```